eng/service.proj

```diff
@@ -2,13 +2,19 @@
   <PropertyGroup>
     <ServiceDirectory Condition="'$(ServiceDirectory)' == '' and '$(Scope)' != ''">$(Scope)</ServiceDirectory>
     <ServiceDirectory Condition="'$(ServiceDirectory)' == ''">*</ServiceDirectory>
+    <IncludeTests Condition="'$(IncludeTests)' == ''">true</IncludeTests>
+    <IncludeSrc Condition="'$(IncludeSrc)' == ''">true</IncludeSrc>
   </PropertyGroup>
 
-  <ItemGroup>
-    <ExcludeMgmtLib Include="..\sdk\$(ServiceDirectory)\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj" />
-    <ProjectReference Include="..\sdk\$(ServiceDirectory)\**\*.csproj" Exclude="@(ExcludeMgmtLib)"/>
+  <ItemGroup Condition="'$(PackageDirectory)' == ''">
+    <ExcludeMgmtLib Include="..\sdk\$(ServiceDirectory)\*.Management.*\**\*.csproj;..\sdk\*mgmt*\**\*.csproj;" />
+    <TestProjects Include="..\sdk\$(ServiceDirectory)\*\tests\**\*.csproj" />
+    <SrcProjects Include="..\sdk\$(ServiceDirectory)\**\*.csproj" Exclude="@(TestProjects)" />
+    <ProjectReference Include="@(TestProjects)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeTests)' == 'true'" />
+    <ProjectReference Include="@(SrcProjects)" Exclude="@(ExcludeMgmtLib)" Condition="'$(IncludeSrc)' == 'true'" />
   </ItemGroup>
-  <Import Project="..\sdk\$(ServiceDirectory)\*.projects" />
+
+  <Import Condition="'$(ServiceDirectory)' != ''" Project="..\sdk\$(ServiceDirectory)\*.projects" />
```

Comment:
Why did you need to add the condition on this import?

Reply:
That was from my previous attempt to use the package directory. I have removed that. Thanks for pointing it out.